packages/deploy/src/bridge/BridgeContracts.ts

```diff
@@ -659,55 +655,60 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
       checklist.equals(
         undefined,
         this.data.ethHelper,
-        "ethHelper shouldn't exist",
+        "ethHelper doesn't exist (as expected)",
       );
     }
 
     //  ========= custom tokens =========
 
     if (this.data.customs) {
       for (const custom of this.data.customs) {
-        const addresses = custom.addresses;
-        checklist.assertBeaconProxy(addresses, 'Custom Token');
+        const { name, symbol, addresses } = custom;
+        checklist.assertBeaconProxy(
+          addresses,
+          `Custom Token is deployed for ${name} (${symbol})`,
+        );
 
         if (this.data.bridgeToken) {
           checklist.equalIds(
             addresses.implementation,
             this.data.bridgeToken.implementation,
-            'BirdgeToken implementation',
+            'Custom Token uses normal BridgeToken implementation',
           );
           checklist.notEqualIds(
             addresses.beacon,
             this.data.bridgeToken.beacon,
-            'BirdgeToken beacon',
+            'Custom Token uses *custom* BridgeToken beacon',
           );
         }
 
-        checklist.notEqualIds(
-          custom.controller,
-          core.upgradeBeaconController.address,
-          'upgradeBeaconController',
-        );
-
         const tokenContract = await contracts.BridgeToken__factory.connect(
           utils.evmId(addresses.proxy),
           this.connection,
         );
 
-        const name = await tokenContract.name();
-        const symbol = await tokenContract.symbol();
+        const chainName = await tokenContract.name();
+        const chainSymbol = await tokenContract.symbol();
         const decimals = await tokenContract.decimals();
         const owner = await tokenContract.owner();
 
         checklist.equalIds(
           owner,
           this.bridgeRouterContract.address,
-          'bridgeRouterContract',
+          'Custom Token is owned by BridgeRouter',
```

Comment:
We should either use BridgeRouter, Bridge Router or bridge router. 

Below we use `governance router` and I suggested `Governance Router`.  Let's pick one and use throughout the script.